### PR TITLE
Fix: Add "Did you know that..." to the analyzing screen

### LINKED ISSUE #2440: [Bug] Fun facts could be misunderstood as analysis insights
# 🐛 Bug report

## Description

The process dialog tries to be helpful with short fun facts, like `52% of resources aren't cachable` or `The average load time for mobile sites is 19 seconds over 3G connections.`. As a user looking at this, there is little to no indication that this is an insight generated from my page or not and might be misunderstood.

Options might be making these hints more subtle and adding a "Did you know?"

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #2440

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)
This change add the message "Did you know that..." to the analyzing screen in the browser scanner so people don't get confused by the messages.

### CODE REVIEW

**Comment by molant** [comment]
@sarvaje can you please add a screenshot here for posterity?

**Comment by sarvaje** [comment]
Sure, I did the screenshot, but I forgot to paste here hahaha.

image

File: packages/extension-browser/src/devtools/views/pages/analyze.css [hunk 1]
```diff
@@ -41,6 +41,12 @@
     font-size: var(--font-size-sm);
 }
 
+.message-title {
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/pages/analyze.css]: I've been following the [CSS Modules recommendation of camelCased names](https://github.com/css-modules/css-modules#naming) to allow direct object access in our script files.
```suggestion
.messageTitle {
```

File: packages/extension-browser/src/devtools/views/pages/analyze.tsx [hunk 2]
```diff
@@ -109,7 +109,10 @@ const Analyze = ({ config, onCancel, onError, onResults, onTimeout }: Props) =>
                     {getMessage('analyzingStatus')}
                 </h1>
                 <div className={styles.messages}>
-                    <div className={styles.message}>{status}</div>
+                    <div className={styles.message}>
+                        <h2 className={styles['message-title']}>{getMessage('didYouKnowThat')}</h2>
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/pages/analyze.tsx]:
```suggestion
                        <h2 className={styles['message-title']}>{getMessage('didYouKnowThat')}</h2>
```
```suggestion
                        <h2 className={styles.messageTitle}>{getMessage('didYouKnowThat')}</h2>
```

**Comment by sarvaje** [comment]
@antross done!

**Pull request merged**